wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/RuntimeTarget.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.workspace.server.model.impl;
+
+import java.util.Objects;
+import org.eclipse.che.api.core.model.workspace.runtime.RuntimeIdentity;
+import org.eclipse.che.commons.annotation.Nullable;
+import org.eclipse.che.commons.subject.Subject;
+
+public final class RuntimeTarget {
```

Comment:
Please add a few lines that would explain what is `runtime target`. For me, it's not obvious...
I understand targetNamespace as namespace where workspace is run...
And I assume that RuntimeTarget is something like who owns runtime, to whom this runtime belongs to...

Reply:
Hopefully the the javadoc makes it clearer now.

Reply:
Sorry, for me `RuntimeTarget` is not clear yet... Why `ownerId` is `RuntimeIdentity` and ownerName is part of `RuntimeTarget`?

Reply:
is not actual anymore